lib/api/corbado_service.dart

```diff
@@ -1,118 +1,37 @@
-import 'dart:convert';
+import 'dart:io';
```

Comment:
shall we rename the "api" directory to "services"? To me this seems to be the more common structure.